OpenProblemLibrary/Hope/Multi1/03-05-Basis-subspace/Basis_30.pg

```diff
@@ -47,6 +40,47 @@ $rrefA = Matrix([
 [0,0,0,0]
 ]);
 
+
+$P = Matrix([[1,0,0], [0,1,0], [0, 0, 1]]);
+
+$paddedA = $P*$rrefA;
```

Comment:
I don't see these being used, they should be deleted.

Reply:
> This works for me. Based on my testing it accepts all and only the correct answers.
> 
> The typos and instructions mentioned below should be fixed before merging.
> 
> My vote would be to change the subspace questions to PopUps. Since there are other correct answers that could otherwise be typed in the answer blank, I think having an answer blank is misleading. I also think that it is a bad experience for students to be able to type answers that aren't allowed, and only later be told that their answer isn't accepted.

I agree. It seems wordCompletion is useful for problems that involve longer proof sentences, or many choices. Using it for one-word answers seems a bit of an overkill. But I am neutral on this. Does anyone else have any opinion on this?

Reply:
Yes, thanks!